Common/src/geometry/CGeometry.cpp

```diff
@@ -3637,7 +3638,56 @@ const CCompressedSparsePatternUL& CGeometry::GetEdgeColoring(su2double* efficien
 
     /*--- Color the edges. ---*/
     constexpr bool balanceColors = true;
-    edgeColoring = colorSparsePattern(pattern, edgeColorGroupSize, balanceColors);
+
+    /*--- lambda to account for different numbers of colors ---*/
```

Comment:
Just for consistency with the comment style in this function.
```suggestion
    /*--- Lambda to account for different numbers of colors. ---*/
```